examples/02_decoding/plot_oasis_vbm.py

```diff
@@ -60,7 +60,7 @@
 from nilearn.image import get_data
 from nilearn.maskers import NiftiMasker
 
-n_subjects = 100  # more subjects requires more memory
+n_subjects = 50  # more subjects requires more memory
```

Comment:
oops. I understand the concern regarding light models, but we're giving a poor example, doing ML with low n.

Reply:
Good with me: not actually 100% it speeds things up and in any case this is beyond the scope of this PR.

There are a few other places (in previous PRs) where I reduced the number of subjects included in the example: should I plan to revert those?

Reply:
If it's not justified by significant computation time savings, I'd revert if you don't mind.

Reply:
will fix in a separate PR